### DISCUSSION TITLE
UWP无法使用

### DISCUSSION THREAD

**Discussion by Duan-rax** [1715173329/UnblockNeteaseMusic#326] [Ideas]
使用glider 将http代理端口转为ss服务，并成功在安卓和ios上使用

启动命令：pm2 restart app.js --name="UnblockNeteaseMusic" -- -s -p 18123:18125 -o kuwo -e https://music.163.com

日志：|UnblockNeteaseMusic  | [418602085] 英雄
0|UnblockNeteaseMusic  | http://other.player.rh01.sycdn.kuwo.cn/e78468fd323ef59f16a2d0ac731efb49/619133a6/resource/n2/27/63/2871808815.mp3
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > clientlog.music.163.com:443
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125
0|UnblockNeteaseMusic  | TUNNEL > localhost:18125

**Comment by Duan-rax** [comment]
I can use it by http proxy, but if i use ss proxy, it’s not work,It looks like it keeps requesting the https port I set

**Comment by Duan-rax** [comment]
I use the latest code,The following is glider's configuration:
#########
verbose=True

listen=ss://AEAD_AES_128_GCM:password@:18123

forward=http://127.0.0.1:18123
#########

**Comment by 1715173329** [comment]
I have no idea how glider works.
Are you using the same port for UNM and SS instance?
